docs/install.md

```diff
@@ -61,10 +74,15 @@ secret is created and available before spawning the **aws-load-balancer-controll
     aws-load-balancer-operator-controller-manager-b55ff68cc-85jzg   2/2     Running   0          3h26m
     ```
 
-### Use predefined `CredentialsRequest`
-In case the provisioning of the credentials secret should not be done by the **cloud-credential-operator**, the secret needs to be explicitly referenced in `AWSLoadBalancerController` CR, see [credentials.name field description](./tutorial.md#credentialsname).    
-However, this credentials secret needs to reference a role with all the policies needed by the controller. For this purpose a dedicated controller's `CredentialsRequest` is maintained in [hack/controller](../hack/controller/) directory of this repository.
-Its contents are identical to the ones requested from the **cloud-credential-operator**.
+#### Use a Predefined Secret
+In some cases, the provisioning of the credentials secret cannot be done by the **cloud-credential-operator**, e.g. ROSA user clusters are forbidden to create `CredentialsRequest`.
+Instead, the secret can be explicitly referenced in the `AWSLoadBalancerController` CR, see [credentials.name field description](./tutorial.md#credentialsname).
+The credentials secret needs to reference a role with all the policies needed by the controller.
+There are two options for creating the credentials secret.  Use Option 1 with `ccoctl`, or Option 2 with the AWS CLI.
+
+##### Option 1. Using `ccoctl`
+The dedicated controller's `CredentialsRequest` is maintained in [hack/controller](../hack/controller/) directory of this repository.
+Its contents are identical to the ones requested by **aws-load-balancer-operator** from the **cloud-credential-operator**.
```

Comment:
You need to add the ccoctl step here too.

1. Extract and prepare the `ccoctl` binary as described in [Configuring the Cloud Credential Operator utility](https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts).

Reply:
Done.